doc/topics/development/deprecations.rst

```diff
@@ -45,13 +45,13 @@ Consider the following example:
     def some_function(bar=False, foo=None):
         if foo is not None:
             salt.utils.versions.warn_until(
-                "Aluminum",
+                "Aluminium",
                 "The 'foo' argument has been deprecated and its "
                 "functionality removed, as such, its usage is no longer "
                 "required.",
             )
 
-Development begins on the ``Aluminum`` release when the ``3002`` branch is
-forked from the develop branch.  Once this occurs, all uses of the
-``warn_until`` function targeting ``Aluminum``, along with the code they are
+Development begins on the ``Aluminium`` release when the ``3002`` branch is
```

Comment:
Since SEP 14 and the new branching and releasing strategy this sentence isn't accurate: "Development begins on the [sic] XX release when the [sic] xx branch is forked from the [sic] whatever branch."  

Now we are always in development, and I suggest this needs rewording. The `warn_until` is still valid to use since the Periodic Table code names are still in place and we keep our cadence of releases to every 3 months is a major release followed by a minor release.

Reply:
Made some modifications to the wording. How's it look?